addons/OPT/REVIVE/fn_clientInitEH.sqf

```diff
@@ -0,0 +1,82 @@
+/**
+* Description:
+* 
+* 
+* Author:
+* Lord-MDB
+*
+* Arguments:
+*
+* Return Value:
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* call FUNC(clientInitEH);
+*/
+
+#include "macros.hpp";
+
+DUMP("REVIVE EH");
+
+//Event Aüslösung bei bewustlosen Spieler.
+[
+	"ace_unconscious",
+	{
+		params ["_unit", "_isUnconscious"]; 
+
+		[_unit] call FUNC(isUnconscious);
+	}
+] call CBA_fnc_addEventHandler;
+
+//Funktion für EH auslösung
+DFUNC(isUnconscious) = 
+{
+	params ["_unit"];
+
+	if (_unit isEqualTo player) then 
+	{
+		//Var für GPS setzen 
+		_unit setVariable ["FAR_isUnconscious", 1, true];
```

Comment:
In einer Funktion, die `isUnconscious` heißt, finde ich es fatal, wenn da irgendwelche Variablen gesetzt werden. Rein vom Namen, sollte soclh eine Funktion absolut keine Nebeneffekte mit sich bringen.

Das gilt auch für die weiteren Aktionen, die in dieser Funktion getätigt werden.